k8s/helmfile/argo-cd.yaml

```diff
@@ -1,37 +1,18 @@
-environments:
-  default:
```

Comment:
issue: I think this block actually needs to remain otherwise even if you (for example) `helmfile apply -e local` you will actually target whatever is the current kube-context